samples/camera/src/main.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2019 NXP Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/camera.h>
+#include <drivers/display.h>
+#include <stdio.h>
+#include <misc/__assert.h>
+
+/* _______          _________
+ *|       |<-------|head|    |
+ *|       |	   |____|    |___________      ______
+ *|	  |	   |tail|<---|__camer_IP_|<---|_CMOS_|
+ *|	  |------->|____|____|
+ *|_______|	   |____|____|
+ * Display	      Camera
+ */
+
+#define CAMERA_DEMO_FB_MAX_SIZE 0x40000
+#define CAMERA_DEMO_FB_MAX_NUM 4
+char gfb[CAMERA_DEMO_FB_MAX_SIZE * CAMERA_DEMO_FB_MAX_NUM] __aligned(64);
+struct k_sem gsem;
+
+static void capture_cb(void *fb, int w, int h, int bpp)
+{
+	k_sem_give(&gsem);
+}
+
+void main(void)
+{
+	int ret, i;
+	struct device *camera_dev = camera_get_primary();
+	struct device *display_dev =
+		device_get_binding(CONFIG_CAMERA_DISPLAY_DEV_NAME);
+	struct display_capabilities diaplay_cap;
```

Comment:
should this be ``display_cap`` and ``display_dsc``?  Seems like a misspelling

Reply:
I don't want the names to be too long.
I see "struct display_capabilities cap;" in Lvgl_color_1.c (lib\gui\lvgl).
How about "display_desc"?

Reply:
I was just commenting that ``diaplay`` in that name should be spelled as ``display``

Reply:
Sorry for this typo that I did't notice:) I will fix it.